npm/react/cypress/component/advanced/timers/card-without-effect.cy.jsx

```diff
@@ -44,7 +45,8 @@ it('should cleanup on being removed (using unmount)', () => {
     expect(onSelect).to.not.have.been.called
   })
 
-  unmount()
+  // mount something else so that unmount is called
+  mount(<div>Test Component</div>)
```

Comment:
Should we use the suggested alternative:
`cy.then(() => ReactDom.unmountComponentAtNode(getContainerEl()))`